src/main/java/org/geysermc/resources/state/type/VinesAgeMapper.java

```diff
@@ -0,0 +1,95 @@
+package org.geysermc.resources.state.type;
+
+import org.geysermc.resources.Pair;
+import org.geysermc.resources.state.StateMapper;
+import org.geysermc.resources.state.StateRemapper;
+
+@StateRemapper(value = "age", blockRegex = ".*_vines.?$")
+public class VinesAgeMapper extends StateMapper<Integer> {
+
+    @Override
+    public Pair<String, Integer> translateState(String fullIdentifier, String value) {
+        int age = 0;
+        switch (value) {

```

Comment:
Wouldn't it be easier to use `Integer.fromString(value)` (may not be exact wording)?

Reply:
Yeah, would be `Integer.parseInt(value)`. Already marked this for review :wink: